ObsLocTAP.tex

```diff
@@ -554,6 +559,28 @@ \subsection{Observation Locator data model }
 removed from the planning table or they could return to the "Planned" status
 for a possible future schedule.
 
+Tracking type (\textbf{tracking\_type}) is an optional field of the obsplan table
+to characterize the tracking foreseen during the observation execution.  
+The default value would be \textbf{RA/Dec}. 
+For this observations the pointing is mantained on the same position on the sky
+during all the observation period. This kind of tracking is usually known as
+siderial tracking for ground based observations. 
+
+Other possible value is \textbf{solar-system-object-tracking}, to be used when
+the pointing of the observatory varies to maintain the selected solar system
+target on the same position of the observatory point of view. In this case, the
+values for \textbf{s\_ra} and \textbf{s\_dec} could be defined as null or,
+if possible, a representative position within the expected siderial movement.
+In this case, \textbf{target\_name} would be an identifier of the solar system
+object.
+
+The third possible value is \textbf{transit-mode}, to be used when the
+observatory points to a fixed position on the sky where a certain moving target
+will cross the field of view. \textbf{s\_ra} and \textbf{s\_dec} would
+content the value of the fixed position and \textbf{target\_name} would be
+an identifier of the object to be observed.
+
```

Comment:
I propose the following wording:
```latex
The third possible value is \textbf{fixed-az-el-transit}, to be used when the
observatory points to a fixed position in local azimuth-elevation coordinates,
allowing for a target to cross the field of view. \textbf{s\_ra} and \textbf{s\_dec} 
would contain the value of the sidereal pointing at the time of the transit of 
the named target in \textbf{target\_name}, which would be an identifier of 
the object to be observed.
```

I prefer to have rather well defined `tracking_type` so that we can add other 
terms when other use cases are proposed.

Reply:
It looks a lot clearer now. Thanks